_clanky/série-dobrá-praxe-data-královéhradeckého-kraje-a-interoperabilita.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: post
+detail: true
+title: Série Dobrá praxe: Data Královéhradeckého kraje a interoperabilita
```

Comment:
Když název obsahuje dvojtečku, musí být celý název v uvozovkách - jinak se to rozbije, protože dvojtečka odděluje klíč a hodnotu v YAML - což je tady ta hlavička
```suggestion
title: "Série Dobrá praxe: Data Královéhradeckého kraje a interoperabilita"
```